store/src/main/java/org/apache/rocketmq/store/MappedFileQueue.java

```diff
@@ -191,7 +199,7 @@ public long howMuchFallBehind() {
         return 0;
     }
 
-    public MappedFile getLastMappedFile(final long startOffset, boolean needCreate) {
+    private MappedFile getLastMappedFile(final long startOffset, boolean needCreate) {
```

Comment:
Yes,  some methods in `MappedFileQueue` may have their access modifier narrowed down in grammatical, but I think these methods should be `public`.